src/PlaywrightSharp/build/netstandard2.0/PlaywrightSharp.props

```diff
@@ -5,15 +5,15 @@
   so copy the native assemblies to the output directory.
   -->
 	<ItemGroup Condition="Exists('packages.config') OR Exists('$(MSBuildProjectName).packages.config') OR Exists('packages.$(MSBuildProjectName).config')">
-		<Content Include="$(MSBuildThisFileDirectory)\..\..\runtimes\win-x64\native\**" Condition="'$(PlatformTarget)' != 'x86'">
+		<Content Include="$(MSBuildThisFileDirectory)\..\..\.playwright\win-x64\native\**" Condition="'$(PlatformTarget)' != 'x86'">
 			<CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
 			<Visible>false</Visible>
 			<Link>%(RecursiveDir)%(Filename)%(Extension)</Link>
 		</Content>
-		<Content Include="$(MSBuildThisFileDirectory)\..\..\runtimes\win-x86\native\**" Condition="'$(PlatformTarget)' == 'x86'">
+		<Content Include="$(MSBuildThisFileDirectory)\..\..\.playwright\win-x86\native\**" Condition="'$(PlatformTarget)' == 'x86'">
```

Comment:
This should be drivers because it's taking it from the nuget

Reply:
That makes sense.